client/src/components/pages/Modeler/Modeler.jsx

```diff
@@ -3,21 +3,26 @@ import { Layout } from 'antd';
 import BpmnModeler from '../../content/BpmnModeler/BpmnModeler';
 import HeaderNavbar from '../../content/HeaderNavbar/HeaderNavbar';
 
+
 const { Footer } = Layout;
 
 /**
  * @description Modeler page that enables the user to build a robot
  * @category Client
  * @component
  */
-const Modeler = () => (
-  <div>
-    <Layout>
-      <HeaderNavbar selectedKey={2} />
-      <BpmnModeler />
-      <Footer>Fußzeile</Footer>
-    </Layout>
-  </div>
-);
+const Modeler = (match) => {
+  const { robotId } = match.match.params;
```

Comment:
What does this do? :)

Reply:
This piece of code retrieves the passed RobotId (which was passed via the routing of the React router).

`RobotContainer.jsx` (the edit-Button links to the modeler)
![image](https://user-images.githubusercontent.com/36270527/110843229-55c4a780-82a8-11eb-8be7-78f0d4521667.png)

the final `route` is shown in the `URL`
![image](https://user-images.githubusercontent.com/36270527/110842916-ff576900-82a7-11eb-9b78-b80b24b7473d.png)

`App.js` (responsible for all routes)
![image](https://user-images.githubusercontent.com/36270527/110843037-22821880-82a8-11eb-943f-7c6ac9bee305.png)

If you want more information about routing with params, have a look at [this article](https://betterprogramming.pub/how-to-pass-multiple-route-parameters-in-a-react-url-path-4b919de0abbe) [section "Routing Parameters"]. @SanJSp @WolfgangDaniel

Reply:
pretty cool :) thanks!